osp-backend-docusaurus/docs/Getting-Started.md

```diff
@@ -0,0 +1,35 @@
+---
+id: Getting-Started
+title: Getting Started
+---
+
+Hello Newcomer, here are some tips to help you get started contributing to this project.
+
+## Get to know the project
+Here's what you can do to know more about the project:
+
+* Read the documentation available on this [Docusaurus website](https://osp-backend-docs.surge.sh/) and [README](https://github.com/anitab-org/open-source-programs-backend/blob/develop/README.md).
+* You can join the [#open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs) channel on AnitaB.org Open Source Zulip, to see or participate in the project's discussion.
+* You can browse the code on GitHub or even on your workspace after cloning it.
+
+### Choose a working item
+* Check the available issues (that anyone can contribute to) or first timers only issues (just for first-time contributors in this project).
+* Choose one issue you want to work on.
+* Ask maintainers, on the issue's comment section, if you can work on it.
+* Once you get the approval you can start working on it.
+
+### Start working
+Before you start working check the [Contribution Guidelines](https://github.com/anitab-org/open-source-programs-backend/blob/develop/.github/CONTRIBUTING.md) to make sure you can follow the best practices.
+
+**In short:**
+
+1. Fork the project into your profile.
```

Comment:
You can add links to `[Fork](./Fork,-Clone,-Remote-and-Pull-Request.md#Fork)`, similarly to `Clone` and `Setup remotes`.